app/build.gradle.kts

```
@@ -97,8 +113,8 @@ dependencies {
 
     // Cast
     implementation(Dependencies.Cast.mediaRouter)
-    implementation(Dependencies.Cast.playServicesCast)
-    implementation(Dependencies.Cast.playServicesCastFramework)
+    add("proprietaryImplementation", Dependencies.Cast.playServicesCast)
+    add("proprietaryImplementation", Dependencies.Cast.playServicesCastFramework)
```

I think you should be able to get a reference to an actual function instead of a string here..
Something like 
```suggestion
    val proprietaryImplementation by configurations
    proprietaryImplementation(Dependencies.Cast.playServicesCast)
    proprietaryImplementation(Dependencies.Cast.playServicesCastFramework)
```

---

Unfortunately not

---

With `configurations.creating` maybe?

---

Either way, see this: https://guides.gradle.org/migrating-build-logic-from-groovy-to-kotlin/#configurations-and-dependencies

---

Thanks, fixed now! :)